commcare_export/commcare_minilinq.py

```diff
@@ -224,7 +225,8 @@ class DatePaginator(SimplePaginator):
 
     DEFAULT_PARAMS = object()
 
-    def __init__(self, since_field, page_size=1000, params=DEFAULT_PARAMS):
+    def __init__(self, since_field, page_size=None, params=DEFAULT_PARAMS):
+        page_size = page_size if page_size else 1000
```

Comment:
Is there a reason why the default value for `page_size` was removed and was instead put into this conditional check?

Reply:
Ah, interesting! I never knew about this "gotcha".